app/src/modules/insights/routes/panel-configuration.vue

```diff
@@ -112,103 +141,257 @@ const stageChanges = () => {
+									/>
+								</div>
+
+								<div class="field half-right">
+									<p class="type-label">{{ t('color') }}</p>
+									<interface-select-color
+										:value="panel.color"
+										:disabled="panel.show_header !== true"
+										width="half"
+										@input="edits.color = $event"
+									/>
+								</div>
+
+								<div class="field full">
+									<p class="type-label">{{ t('note') }}</p>
+									<v-input
+										:model-value="panel.note"
+										:disabled="panel.show_header !== true"
+										:placeholder="t('panel_note_placeholder')"
+										@update:model-value="edits.note = $event"
+									/>
+								</div>
+							</div>
+						</div>
+					</div>
+				</transition-expand>
 			</div>
 		</div>
 	</v-drawer>
 </template>
 
-<style scoped>
+<style scoped lang="scss">
 .content {
 	padding: var(--content-padding);
 	padding-top: 0;
 	padding-bottom: var(--content-padding-bottom);
 }
 
-.select {
-	margin-bottom: 32px;
+.v-divider {
+	margin: 68px 0 48px;
 }
 
-.panel-type-label {
-	margin-bottom: 16px;
+.group h2 {
+	margin-bottom: 40px;
+	padding-bottom: 2px;
+	font-weight: 700;
+	border-bottom: var(--border-width) solid var(--border-subdued);
 }
 
-.v-divider {
-	margin: 68px 0 48px;
+.group + .group {
+	margin-top: 80px;
+}
+
+.panel-grid {
+	--columns: 1;
+
+	display: grid;
+	grid-template-columns: repeat(var(--columns), 1fr);
+	gap: 32px;
+
+	@media (min-width: 400px) {
+		--columns: 2;
+	}
+
+	@media (min-width: 600px) {
+		--columns: 3;
+	}
+
+	@media (min-width: 840px) {
+		--columns: 4;
+	}
+}
+
+.interface {
+	min-height: 100px;
+	overflow: hidden;
+	text-align: left;
+}
+
+.preview {
+	--v-icon-color: var(--background-page);
+
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 160px;
+	height: 100px;
+	margin-bottom: 8px;
+	border: var(--border-width) solid var(--border-subdued);
+	border-radius: var(--border-radius);
+	transition: var(--fast) var(--transition);
+	transition-property: background-color, border-color;
+}
+
+.preview img {
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
+}
+
+.preview .svg {
+	display: contents;
+}
+
+.preview :deep(svg) {
+	width: 100%;
+	height: 100%;
+}
+
+.preview :deep(svg) .glow {
+	filter: drop-shadow(0 0 4px var(--primary-50));
+}
+
+.preview .fallback {
+	--v-icon-color: var(--primary-75);
+
+	display: block;
+	padding: 8px 16px;
+	background-color: var(--background-page);
+	border: 2px solid var(--primary);
+	border-radius: var(--border-radius);
+	box-shadow: 0 0 8px var(--primary-75);
+}
+
+.interface:hover .preview {
+	border-color: var(--border-normal);
+}
+
+.interface.active .preview {
+	background-color: var(--primary-alt);
+	border-color: var(--primary);
+}
+
+.interface.gray .preview {
+	--primary: var(--gray);
+	--primary-20: var(--gray-20);
+	--primary-30: var(--gray-30);
+	--primary-40: var(--gray-40);
+	--primary-50: var(--gray-50);
+	--primary-70: var(--gray-70);
+	--primary-80: var(--gray-80);
```

Comment:
Lets update the SVGs to use the theme--primary colors instead to make sure this remains long-term compatible